README.md

```diff
@@ -22,6 +22,36 @@ Every ID (except systemId, which (by GBFS convention) should already be globally
 At least every operator should be assigned its own unique codespace, assuming operators already maintain data / ID separation accross their feeds.
 The codespaced operatorId should comply with the following format `<codespace>:Operator:<operator>`.
 
+#### Configuring Authentication
+
+Some providers may require authentication, e.g. via bearer token, OAuth2 or custom http-headers.
+
+    lamassu:
+        providers:
+          - systemId: my_oauth2_secured_system
+            authentication:
+                scheme: OAUTH2_CLIENT_CREDENTIALS_GRANT
+                properties:
+                    tokenUrl: "https://mytokenurl.example/"
+                    clientId: my-client-id
+                    clientPassword: my-client-password
+            ...
+          - systemId: my_bearer_secured_system
+            authentication:
+                scheme: BEARER_TOKEN
+                properties:
+                    accessToken: my-access-token
```

Comment:
I think it's a bad practice to add unencrypted credentials to config files. I know this might be controversial, but I think we should only demonstrate how to pass them in via env vars.

Reply:
While I agree with you in principle, I have been quite liberal about this in our own setup since, frankly, I'm not dealing with any sensitive data, and it should be public anyway. That said, I don't really mind if it isn't demonstrated in the docs.

Reply:
Added an explicit note recommending not to store credentials in plain text.